test/common/EdgeDevice.cs

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+namespace common
+{
+    using System;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Devices;
+
+    public class EdgeDevice
```

Comment:
I think we should merge EdgeDaemon into EdgeDevice.

Reply:
Compare the surface of EdgeDevice:
```
ConnectionString
Id
CreateIdentityAsync()
GetOrCreateIdentityAsync()
DeleteIdentityAsync()
MaybeDeleteIdentityAsync()
```
to that of EdgeDaemon:
```
InstallAsync()
UninstallAsync()
StartAsync()
StopAsync()
```

EdgeDevice is about the device's identity in the cloud. EdgeDaemon is about managing the local installation of our software. I think I get why you would want to merge them conceptually, but practically I don't it's the right approach.

Reply:
ok.